CHANGES

```diff
@@ -1,5 +1,56 @@
 This file describes the most significant changes. For more detail, use
 'git log' on a clone of the charm repository.
+================================================================================
+What's new in Charm++ 8.0.0
+================================================================================
+
+This is a feature release, with the following major changes:
+
+Highlights:
+
+- The License has changed to Apache 2.0 with LLVM exception.  This
+  change to a popular Open Source license is intended to simplify use,
+  collaboration, and greater community involvement in the development
+  of Charm++. The NOTICE file contains the pertinent disclaimers.
+
+- The CkIO library (previously only supporting file output) has been
+  enhanced to support file input. The input layer enables two-phase,
+  collective input from a single file via the an array of buffer chares,
```

Comment:
via ~~the~~ an array

Reply:
@mayantaylor Please make the suggested change